README.md

```diff
@@ -174,45 +174,45 @@ Support for OpenGL accelerated rendering is made possible through the work our D
 
-To enable rich integration in WSLg, we've also added a small plug-in to the RDP backend specific to WSLg. In Weston, the plug-in is responsible for attaching to the user distro and searching for installed application (aka desktop file). The plug-in sends the Windows host a list of all  applications found, along with the command line to launch them and their icon. On the Windows host, an open source [mstsc plugin](https://github.com/microsoft/wslg/tree/main/WSLDVCPlugin) part of the WSLg project uses that information to create shortcuts for these Linux applications to the Windows Start Menu.
+To enable rich integration in WSLg, we've also added a small plugin to the RDP backend specific to WSLg. In Weston, the plugin is responsible for attaching to the user distro and searching for installed applications (aka the desktop file). The plugin sends the Windows host a list of all applications found along with their launch commands and icons. On the Windows host, an open source [mstsc plugin](https://github.com/microsoft/wslg/tree/main/WSLDVCPlugin) part of the WSLg project uses that information to create shortcuts for these Linux applications to the Windows Start Menu.
 
-We've also fix several bugs impacting various applications. Generally these were problem that impacted Weston in all modes and not specific to WSLg.
+We've also fixed several bugs impacting various applications. Generally, these were problems that impacted Weston in all modes (and not specific to WSLg).
 
 ## FreeRDP
-Weston currently uses FreeRDP for its RDP Backend. WSLg continues to leverage FreeRDP and we have added support for new RDP Protocol/Channel to enable VAIL optimized scenario as well as support for the WSLg plugin. We've also fixed various bugs that were impacting interop with mstsc or causing instability.
+Weston currently uses FreeRDP for its RDP Backend. WSLg continues to leverage FreeRDP and we have added support for a new RDP Protocol/Channel to enable VAIL optimized scenario as well as support for the WSLg plugin. We've also fixed various bugs that were impacting interops with mstsc or causing instability.
 
 ## PulseAudio
-For PulseAudio, our contribution is focused on both a sink and a source plug-in which shuffle audio data between PulseAudio and the Weston RDP backend such that the audio data can be integrated over the RDP connection back to the host. There are no changes to the core of PulseAudio itself outside of adding these new plug-in.
+For PulseAudio, our contributions focused on a sink and a source plugin that shuffle audio data between PulseAudio and the Weston RDP backend such that the audio data can be integrated over the RDP connection back to the host. There are no changes to the core of PulseAudio outside of adding these new plugins.

```

Comment:
```suggestion
For PulseAudio, our contributions are focused on a sink and a source plugin that shuffle audio data between PulseAudio and the Weston RDP backend such that the audio data can be integrated over the RDP connection back to the host. There are no changes to the core of PulseAudio outside of adding these new plugins.

```

Reply:
The contributions focused on those 2 elements, adding "are" sounds awkward.

Reply:
Oh, it's talking about the past contributions, or the the past, current, and future contributions?

Reply:
Pretty sure just past.

Reply:
Okay, then I'd undo (or not apply) this change 👍🏻